Training/Training/Training.c

```diff
@@ -23,45 +23,36 @@
 #define ANSI_COLOR_RED     "\x1b[31m"
 #define ANSI_COLOR_GREEN   "\x1b[32m"
 #define ANSI_COLOR_RESET   "\x1b[0m"
-#define ANSI_COLOR_YELLOW  "\x1b[33m"
 
-/// <summary>
-/// Function which checks if a given string is palindrome or not
-/// </summary>
+/// <summary>Function which checks if a given string is palindrome or not</summary>
 int PalindromeChecker (char* phrase) {
-   int i = 0, j = 0, length, newLength, flag = 0;
+   int i = 0, j = 0, length;
    char temp[MAX];

```

Comment:
There should be no IntelliSense warning in your program. Here it is giving a warning on uninitialized variable.